ggshield/cmd/common_options.py

```diff
@@ -140,13 +140,25 @@ def log_file_callback(
 )
 
 
+_exit_zero_option = click.option(
+    "--exit-zero",
+    is_flag=True,
+    default=None,
+    envvar="GITGUARDIAN_EXIT_ZERO",
+    help="Always return a 0 (non-error) status code, even if incidents are found."
+    "The env var GITGUARDIAN_EXIT_ZERO can also be used to set this option.",
+    callback=create_config_callback("exit_zero"),
+)
+
+
 def add_common_options() -> Callable[[AnyFunction], AnyFunction]:
     def decorator(cmd: AnyFunction) -> AnyFunction:
         _verbose_option(cmd)
         _debug_option(cmd)
         _log_file_option(cmd)
         _allow_self_signed_option(cmd)
         _check_for_updates(cmd)
+        _exit_zero_option(cmd)
```

Comment:
This changes the behaviour because it will be added to all the commands. I am not sure that we want to do this.
We should declare the `_exit_zero_option` here but actually use it only where it was used before.